app/helpers/admin/claims_helper.rb

```diff
@@ -50,5 +51,21 @@ def display_school(school)
       ].join(" ").html_safe
       sanitize(html, tags: %w[span a], attributes: %w[href class])
     end
+
+    def deadline_warning(claim)
+      return unless claim.approaching_deadline? || claim.passed_deadline?
+      content_tag(
+        :strong,
+        "#{claim.passed_deadline? ? "- " : ""}#{pluralize(weekdays_between(Date.today, claim.deadline_date), "day")}",
+        class: "govuk-tag claim-deadline__#{claim.passed_deadline? ? "passed" : "approaching"}"
+      )
+    end
+
+    private
+
+    def weekdays_between(first_date, second_date)
```

Comment:
What if this method was updated to return a negative value when the second date was before the first date? Would that simplify the `deadline_warning` method?